### ISSUE TITLE
The example installation instructions instruct the users to build on master

### ISSUE THREAD

**Issue by murgatroid99** [grpc/grpc.github.io#105]
This site is based on the content of the release branch, but the installation instructions for the examples say to clone master and then run the installation. After the clone step, there should be a `git checkout origin/release-0_11` step.

**Comment by LisaFC** [comment]
Yeah, it looks there are a few orphan (and very old) pages still being served on the site - they're not linked to anything but they're still there. I'll remove them.

**Comment by jtattermusch** [comment]
@murgatroid99  @LisaFC  has this been addressed?

**Comment by ejona86** [comment]
This looks mostly fixed. It seems only the C++ quick start needs to be fixed (the tutorial is fine). I also noticed the csharp tutorial (quick start is fine) says `-b v1.0.x`. It's probably not using the expression we use elsewhere and needs to be fixed.
